internal/services/v1/experimental.go

```diff
@@ -206,3 +210,151 @@ func (es *experimentalServer) BulkLoadRelationships(stream v1.ExperimentalServic
 		NumLoaded: numWritten,
 	})
 }
+
+func (es *experimentalServer) BulkExportRelationships(
+	req *v1.BulkExportRelationshipsRequest,
+	resp v1.ExperimentalService_BulkExportRelationshipsServer,
+) error {
+	ctx := resp.Context()
+	ds := datastoremw.MustFromContext(ctx)
+
+	var atRevision datastore.Revision
+	var cur dsoptions.Cursor
+
+	if req.OptionalCursor != nil {
+		decoded, err := cursor.Decode(req.OptionalCursor)
+		if err != nil {
+			return shared.RewriteError(ctx, err)
+		}
+
+		atRevision, err = ds.RevisionFromString(decoded.GetV1().Revision)
+		if err != nil {
+			return shared.RewriteError(ctx, err)
+		}
+
+		cur = &core.RelationTuple{
```

Comment:
I'd recommend just encoding the string form of the tuple into the cursor and using `tuple.MustParse`